packages/@webex/plugin-meetings/src/locus-info/index.ts

```diff
@@ -512,6 +513,14 @@ export default class LocusInfo extends EventsScope {
   updateLocusFromHashTreeObject(object: HashTreeObject, locus: LocusDTO): LocusDTO {
     const type = object.htMeta.elementId.type.toLowerCase();
 
+    const addParticipantObject = (obj: HashTreeObject) => {
+      if (!locus.participants) {
+        locus.participants = [];
+      }
+      locus.participants.push(obj.data);
```

Comment:
the body of addParticipantObject() is just code moved from lines 580-586, but it's simplified a bit, because `object.data` actually always contains a copy of `htMeta` - the same one we have in `object.htMeta` so we don't need to copy it